releases/release-1.27/release-notes/maps/pr-115346-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 115346
+releasenote:
+  text: Now device stage path is set whenever available for expansion during mount
```

Comment:
I wonder if this should call out on it being ` node_stage_path`?

Reply:
I've updated it to 
``` `node_stage_path` is now set whenever available for expansion during mount```